rpcs3/Emu/Cell/lv2/sys_net.cpp

```diff
@@ -1595,12 +1610,141 @@ error_code _sys_net_write_dump(ppu_thread& ppu, s32 id, vm::cptr<void> buf, s32
 	return CELL_OK;
 }
 
+error_code lv2_socket::handle_abort(s32 type, u64 arg, s32 flags)
```

Comment:
From a pure design pov I feel the case _socket should be what is in there, the rest should be in the sys_net_abort function. lv2_socket is meant to be an interface for functions specifically acting on a socket.